package.json

```diff
@@ -16,7 +16,7 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.3.5",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.

Reply:
Got it. I've already made some more "git commit -am" pushes but will add only src files in the future.